pages/DSNP/Announcements.md

```diff
@@ -4,18 +4,16 @@ name: Announcements
 route: /DSNP/Announcements
 ---
 
-
-# TODO
-
-I took out signatures from this level. Do we still need them for DSNP 1.0?
-
-
 # Announcements Overview
 
 Announcements are content or reference to content that communicate new user activity to the rest of the network.
 Announcements are associated with an [Identifier](/DSNP/Identifiers) which can be validated as creator of the announcement.
 Depending on the implementation Announcements may be published directly to the network, included in [Batch Publication Files](/DSNP/BatchPublications), or some combination of those two.
 
+## Announcement Validation
+
+[Signature](/DSNP/Signatures) is the first step in validating an announcement.
```

Comment:
I think we will need to expand this. Perhaps something along these lines?

```markdown
There is no guarantee at time of creation that a given announcement will be from the `fromId` claimed in the announcement.
The reader of the announcement MUST perform a validation at read time to ensure authenticity.
Implementations MUST provide a way to validation what [signatures](/DSNP/Signatures) are associated with a given [identifier](/DSNP/Identifiers).
```

Hmm... I like how that is readable, but I wonder if we need something closer to how we do `Ordering Announcements`?

Thoughts @saraswatpuneet?

Reply:
I like it better, let me use those lines